scripts/docker/Dockerfile.sycl

```diff
@@ -47,17 +47,25 @@ RUN wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRO
     | tee /etc/apt/sources.list.d/oneAPI.list
 
 RUN apt-get update && apt-get install -y \
-        intel-oneapi-compiler-dpcpp-cpp-2025.0 \
+        intel-oneapi-compiler-dpcpp-cpp-2025.2 \
         curl \
         && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN curl -LOJ "https://cloud1.cees.ornl.gov/download/oneapi-for-nvidia-gpus-2025.0.0-cuda-12.0-linux.sh" && \
-    chmod +x oneapi-for-nvidia-gpus-2025.0.0-cuda-12.0-linux.sh && \
-    ./oneapi-for-nvidia-gpus-2025.0.0-cuda-12.0-linux.sh
+RUN cd /tmp && git clone https://github.com/intel/llvm.git -b v6.2.0 --depth=1 && cd llvm && \
+    cmake -S unified-runtime -B build_ur \
+      -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ \
+      -DUR_BUILD_TESTS=OFF \
+      -DUR_ENABLE_TRACING=ON \
+      -DUR_BUILD_ADAPTER_CUDA=ON \
+      -DUR_BUILD_ADAPTER_OPENCL=ON \
+      -DCMAKE_BUILD_TYPE=Release && \
+    cmake --build build_ur -j 8 && \
+    cp -r build_ur/lib/* /opt/intel/oneapi/compiler/2025.2/lib/ && \
```

Comment:
Is that really how we are supposed to install the library?  Is there no install target?

Reply:
It looks like there may be install targets indeed:

- https://github.com/intel/llvm/blob/ac28a83267064524560e3abbbda2ab5bf244e364/sycl/cmake/modules/BuildUnifiedRuntime.cmake#L205
- https://github.com/oneapi-src/unified-runtime/blob/4b5c8af50a03cb01b4331766b2efe224563b5183/CMakeLists.txt#L22

Reply:
> Can you share a link for reference?

No, I don't have a link.

Reply:
> should we set the std?

The project states that it needs C++17 and has https://github.com/intel/llvm/blob/faa61805224a993f8f9de6214ba0678afc9e53e4/unified-runtime/CMakeLists.txt#L93-L94. I don't see a reason to specify anything else ourselves.

Reply:
> > Can you share a link for reference?
> 
> 
> 
> No, I don't have a link.

Was it something one of your contact at intel recommended? Where are these instructions coming from?